cookbook/product_asset/sftp.rst

```diff
@@ -0,0 +1,91 @@
+How to connect to an external server for storage
+================================================
+
+Quick Overview
+--------------
+
+**These cookbook examples are made for asset catalog storage but it works the same way for other catalogs**
```

Comment:
**This cookbook will demonstrate you how to store the assets on a SFTP instead of the defaut local filesystem. Assets are specific files availables only in the Enterprise Edition. But you can apply the exact same cookbook to store your product medias elsewhere.**